meta-ov/recipes-apps/ovmenu-ng/files/ovmenu-ng.sh

```diff
@@ -337,19 +337,25 @@ function download_igc_files() {
 }
 
 # Copy XCSaor and OpenVario settings to /usb/usbstick/openvario/backup/<MAC address of eth0>
-function download_files() { 
+function backup_files() { 
 	/usr/bin/backup-system.sh >> /tmp/tail.$$ &
 	dialog --backtitle "OpenVario" --title "Result" --tailbox /tmp/tail.$$ 30 50
 }
 
+# Copy usb/usbstick/openvario/upload/xcsoar to /home/root/.xcsoar
+function upload_xcsoar() { 
+	/usr/bin/upload-xcsoar.sh >> /tmp/tail.$$ &
+	dialog --backtitle "OpenVario" --title "Result" --tailbox /tmp/tail.$$ 30 50
+}
+
```

Comment:
This commit not only rearranges menu items, but also adds a reference to this script, which doesn't (yet) exist?

Reply:
yes, we still have to add it to transfer-xcsoar.sh

Reply:
But that's wrong - this commit is broken!